person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, id = 0, name = 'unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] I'll suggest you generate randomly the id by using the method `rand`.  So, you'll don't need to initialize the id in your initialize method. kindly, remove it and then implement the id like this:
Example: `@id = Random.rand(1...1000)` this has a range of 1 up to 1000.